src/web/releases.rs

```diff
@@ -351,7 +337,7 @@ fn get_search_results(
             target_name: row.get(3),
             release_time: row.get(4),
             rustdoc_status: row.get(5),
-            stars: row.get(6),
+            stars: row.get::<_, i64>(6) as i32,
```

Comment:
Why did this change? Do we store an i64 in postgres?

Reply:
Yes, postgres has an `INT` while `stars` is an `i32`, without this there's a runtime error

Reply:
I stand corrected, I was lied to